docs/ref/react.rst

```diff
@@ -325,10 +325,16 @@ for debugging:
 
    import React from 'react';
    import { I18nProvider } from '@lingui/react';
+   import { i18n } from "@lingui/core"
+   import { en } from 'make-plural/plurals'
+
+   i18n.loadLocaleData('en', { plurals: en })
+   i18n.load('en', messages)
+   i18n.activate('en')
 
    const App = ({ language} ) => {
         return (
-            <I18nProvider language={language} missing="🚨">
+            <I18nProvider i18n={i18n}>
```

Comment:
I believe `missing` prop has been removed completely and this whole paragraph can be removed as well.

Reply:
Ah, nevermind, I see it's still in your todo list 👍

Reply:
Yes! :) I'll ping you once finished